### ISSUE TITLE
prosody support

### ISSUE THREAD

**Issue by willstdueswissen** [LiveHelperChat/xmpp-chat#4]
# Not compatible to Prosody
Hello, i have a question.
Is it feasible to make this Plguin compatible with prosody?
Prosody is better in performance and more modular.

**Comment by remdex** [comment]
Yes there is plans in the future to write plugin for Prosody. Any pull request also welcome. It should not be hard, like you can find source of Ejabberd mods in this repository also :)

**Comment by willstdueswissen** [comment]
Okay, i see what i can do
